Python/Image-Steganography/README.md

```diff
@@ -0,0 +1,59 @@
+# Image Steganography
+Steganography is the art and science of embedding secret messages in a cover message or file in such a way that no one, apart from the sender and the intended recipient, suspects the existence of the message. In case of Image Steganography, secret messages are embedded in images.
+
+![flowchart](https://d1jnx9ba8s6j9r.cloudfront.net/blog/wp-content/uploads/2019/01/SteganogarphyModel-528x259.png)
+
+One of the algorithms to do so is known as **Least Significant Bit Insertion**. In this script the following operations are done:
+
+**While Hiding:**
+1. Image and message is read from a text file.
+2. The message is encrypted with a key.
+3. The encrypted message is then embedded with the image.
+
+**While Retrieving:**
+1. Encrypted message is extracted from the image.
+2. Exncrypted message is then decrypted using the previous key.
+
+## Executing the script:
+* First install dependencies using: `pip install -r requirements.txt`
+* To hide run: `python hide.py <image_path> <message_file_path>`
```

Comment:
scripts have inputs with file names whereas readme asks to input file paths. Please review the readme and update.

Reply:
@tejan-singh Scripts work fine with both file paths and file names. In the screenshots I use filenames because those files were on the same directory.

Reply:
Okay then, I will merge.